docs/metrics.md

```diff
@@ -69,6 +114,26 @@ Here's an example of a complete telemetry ping:
 }
 ```
 
+A Redshift schema for the payload:
+
+```js
+local schema = {
+--   column name                   field type   length  attributes   field name
+    {"timestamp",                  "TIMESTAMP", nil,    "SORTKEY",   "Timestamp"},
```

Comment:
I assume ms, but I don't know.  this was all copied from the standard.  It's rendered at the "second" resolution in redshift and I've seen duplicate rows, but the ms might not be the same.